### DISCUSSION TITLE
Delete Mod completely

### DISCUSSION THREAD

**Discussion by eifelt** [lloesche/valheim-server-docker#323] [Q&A]
Hi,

I have to ask for support again. We have a mod running on the server which unfortunately messes up more than it helps. I have deleted the ddl and the cfg from the respective directories. But the mod is still loaded. How can I delete a mod completely from the server ?

Thx for all

**Comment by lloesche** [comment]
Start the container once with `DEBUG_REINSTALL_BEPINEX=true` or `DEBUG_REINSTALL_VALHEIM_PLUS=true` depending on whether you added the mod to BepInExPack or V+.

**Comment by EifelB** [comment]
Thanks for the quick help. I have transferred the line into the stack. Can the debug command stay in the stack or better take it out again?

**Reply by lloesche** [reply]
If you leave it in then BepInExPack will be re-installed on each container (re)start. What it does is, it deletes `/opt/valheim/bepinex` so the container is forced to do a clean install.
I think I'll create an issue to solve this some better way. Problem right now is I can't just delete things missing in the plugins folder as for instance valheimplus.dll will be in the server's plugins folder but not in the config plugins one.

**Reply by EifelB** [reply]
and a few plugins have additional files and directories. But it is an excellent emergency solution. Through the Mce Mod, more and more mods are coming that can be installed on the server side. 

For people like me who don't know much about programming, a visual solution would be best. I'm thinking of the supervisor or something similar. A list of the mods with a checkbox to uninstall them.
